ocp_pipeline/opm.py

```
@@ -185,67 +185,59 @@ def handle_skill_register(self, message: Message):
-                OCPFeaturizer.ocp_keywords.load_entities(csv_path)
-            if samples:
-                OCPFeaturizer.ocp_keywords.register_entity(kw_label, samples)
-            OCPFeaturizer.ocp_keywords.fit()  # update
+        for s in samples:
+            self.ner.add_word(kw_label, s)
 
-            # warm up the featurizer so intent matches faster (lazy loaded)
-            OCPFeaturizer.extract_entities("UNLEASH THE AUTOMATONS")
 
     def handle_skill_keyword_deregister(self, message: Message):
         skill_id = message.data["skill_id"]
```

Ruff (0.11.9)

Local variable `skill_id` is assigned to but never used

Remove assignment to unused variable `skill_id`

(F841)